ds3-autogen-c/src/main/resources/templates/request-templates/AmazonS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "AmazonS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectras3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
Already fixed in a different branch (moved on to work on Amazon S3 Get separate from this pull request)